TemporaryOysters/bangazon-cli
# Updated main menu CLI - can add payment option

##Pull Requests



##Descriptions:

* Please describe what issue this PR solves











##Steps to Test:
git fetch wc-cli-part2
run "python CLI.py"
you should see main menu & be able to walk your way through each number



##Related Issue:



##System configuration.


* Are there 3rd party libraries that need to be installed?


* Any command line utilities to run?
